CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## Version 1.31.1
+* Release date: July 29, 2021
+* Release status: General Availability
+
+## Hotfix Release
+- Fix for [#16436 Database Connection Toolbar Missing](https://github.com/microsoft/azuredatastudio/issues/16436)
+* Bug Fixes
+  * Accessibility bug fixes 
```

Comment:
1. There aren't any accesibility bug fixes in this release
2. The second bullet is just a duplicate of the "Fix for" link above

Please remove/consolidate these